demisto_client/__init__.py

```diff
@@ -17,8 +19,11 @@
     __version__ = 'dev'
 
 
+DEMISTO_HTTP_HEADERS_REGEX_PATTERN = r'^\w+=\w+(,\w+=\w+)*$'
```

Comment:
are we sure we want to limit to \w? The customer's usecase is for authentication, theres no special chars needed?

Reply:
you are right, ill update regex to include special chars as well.

Reply:
we should also support the case where it is ""

Reply:
do you mean empty string?

Reply:
or even " " should ideally not break it

Reply:
why do we need to support this in regex?

Reply:
if someone has a script that looks like 

```
#setting my envVariables
export Demisto_x..= hello
export DEMISTO_HEADERS= 
export ANOTHWR_HEADING=

```
it shouldnt break

Reply:
i didn't add it to the regex cause it does make sense, added it as a separated check to handle empty strings as well.